modules/oe_theme_helper/templates/oe-theme-helper-in-page-navigation.html.twig

```diff
@@ -8,13 +8,15 @@
  * - entity: The rendering object.
  * - title: Title of the navigation side.
  * - items: In-page navigation items.
+ * - content_prefix: Intro content.
  *
  * @see template_preprocess_oe_theme_helper_in_page_navigation()
  *
  * @ingroup themeable
  */
 #}
 {% set _items = [] %}
+{% set content_prefix = '' %}
```

Comment:
if we mark this variable as an available variable then we should not init it here, rather it will come from outsite and we use a internal here with default value
```suggestion
{% set _content_prefix = content_prefix|default('') %}
```